roles/sys/repositories/tasks/os/Debian/2ndquadrant-repository-keys.yml

```diff
@@ -20,10 +20,8 @@
     if [[ ! -d /usr/share/2ndquadrant-repository-keys ]]; then
       for f in /etc/apt/sources.list.d/2ndquadrant-*; do
         if [[ -e "$f" ]]; then
-          curl https://dl.enterprisedb.com/gpg-key.asc | apt-key add - &&
-          (apt-get update || true) &&
-          apt-get install -y 2ndquadrant-repository-keys;
-          break;
+          echo "Unsupported 2ndQuadrant repositories detected";
+          exit 1;
```

Comment:
Perhaps add a little more detail to the error message? Something to suggest that one shouldn't hit it under normal circumstances unless they are using a VERY old version?

Also, the comment right above this task will need a change as we no longer install the 2ndquadrant-repository-keys package

Reply:
I've expanded the message and reduced the comment. What do you think?

Reply:
looks good. Thanks!